src/components/AddressTypeahead.js

```diff
@@ -80,14 +80,14 @@ export default class AddressTypeahead extends Component {
         query={{
           // available options: https://developers.google.com/places/web-service/autocomplete
           key: AppConfig.GOOGLE_API_KEY,
-          language: 'fr', // language of the results
+          language: AppConfig.LOCALE, // language of the results
           types: 'geocode', // default: 'geocode'
         }}
         styles={ styles }
         nearbyPlacesAPI="GoogleReverseGeocoding" // Which API to use: GoogleReverseGeocoding or GooglePlacesSearch
         GoogleReverseGeocodingQuery={{
           // available options for GoogleReverseGeocoding API : https://developers.google.com/maps/documentation/geocoding/intro
-          region: "fr"
+          region: AppConfig.LOCALE
```

Comment:
I don't think this is correct. It will not work for Belgium for example (locale would be `fr` and `flamish`). I guess it will be better to have another setting `AppConfig.COUNTRY_NAME` (which will be somehow be related to the app store country in which the app is distributed)

What do you think?